wagtailpolls/forms.py

```diff
@@ -34,10 +38,10 @@ def recent_vote(self):
     def clean(self):
         recent_vote = self.recent_vote()
         if not self.ip:
-            self.add_error(None, 'Sorry, we were not able to obtain your ip address')
+            self.add_error(None, _('Sorry, we were not able to obtain your ip address'))
         if recent_vote is not None:
             if timezone.now() - recent_vote.time < datetime.timedelta(minutes=settings.VOTE_COOLDOWN):
-                self.add_error(None, 'Sorry, you cannot vote twice in %s minutes' % (settings.VOTE_COOLDOWN))
+                self.add_error(None, _('Sorry, you cannot vote twice in %s minutes') % (settings.VOTE_COOLDOWN))
```

Comment:
You should use [`ungettext`](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/#pluralization) for translations including plurals, although this is already broken in English when `VOTE_COOLDOWN = 1`